design-system/src/custom-properties.ts

```diff
@@ -129,128 +129,9 @@ export const themes = {
     standardInputHeight: '32px',
   },
   dark: {
-    colorPrimary: '#00b39e',
```

Comment:
Now alternate themes will only contain the properties they need to overwrite from the `default` theme.